runtime/linux/proc/process.go

```diff
@@ -19,11 +19,20 @@
 package proc
 
 import (
+	"path/filepath"
+
+	"github.com/containerd/containerd/defaults"
+	"github.com/containerd/containerd/rootless"
 	"github.com/pkg/errors"
 )
 
 // RuncRoot is the path to the root runc state directory
-const RuncRoot = "/run/containerd/runc"
+func RuncRoot() string {
+	if rootless.RunningAsUnprivilegedUser() {
```

Comment:
Can we run this in Init to prevent it from needing to be checked every time. My understanding is this won't change during the lifetime of the daemon.